app/views/harvest_jobs/show.html.erb

```diff
@@ -42,9 +42,11 @@
         <tr>
           <th scope="row">Estimated Records Extracted</th>
           <td>
-            <%=
-            @harvest_job.extraction_definition.per_page * @harvest_job.extraction_job.documents.total_pages
-            %>
+            <% if @harvest_job.extraction_definition.enrichment? %>
+              <%= 1 * @harvest_job.extraction_job.documents.total_pages %>
```

Comment:
Why the `1 * ...`?

Reply:
Fair call! I just swapped the per page number with what I knew it would be but we don't really need too :P